src/App.js

```diff
@@ -96,6 +99,7 @@ const App = () => (
               <UserRolesView />) : (<Redirect to="/Dashboard" />))}
           />
           <Route path="/accommodations" exact component={ViewAccomodation} />
+          <Route path="/Notifications" exact component={Notifications} />
```

Comment:
Notification route should be written in lowercase so that we can keep the same router naming conventions.